db/app.model.js

```diff
@@ -43,4 +43,17 @@ exports.selectAllArticles = (query) =>{
     
     return rows
   })
+}
+
+exports.selectCommentsByArticleID = (article_id) =>{
+  if(typeof article_id === 'number'){
```

Comment:
This does not need to be here. You should have error handlers specifically for psql error codes that will catch this without a custom reject.